pod/video/Encoding_video_model.py

```diff
@@ -272,14 +271,13 @@ def store_json_list_thumbnail_files(self, info_video, video_to_encode):
         list_thumbnail_files = info_video["list_thumbnail_files"]
         first = True
 
-        videodir, created = UserFolder.objects.get_or_create(
-            name="%s" % video_to_encode.slug,
-            owner=video_to_encode.owner,
-        )
-
         for thumbnail_path in list_thumbnail_files:
             if check_file(list_thumbnail_files[thumbnail_path]):
                 if __FILEPICKER__:
+                    videodir, created = UserFolder.objects.get_or_create(
```

Comment:
Pour celui-ci, je ne suis pas certain du changement, car avant il était en dehors de la boucle. il doit y avoir moyen de ne pas le faire à chaque itération du coup.

Reply:
ok, c fait